nss-cohort-37/Trestlebridge-Purple-Parrots
# Cps farming - add chicken house

# Description
User can add chicken and duck houses.
Fixes Issue # 18
## Type of change
Please delete options that are not relevant.
- x New feature (non-breaking change which adds functionality)
# Testing Instructions
Add both a chicken and a duck house, then add a chicken and a duck to the corresponding facilities and press 4 on the main menu to verify with the farm report function.
`git checkout master`
`git fetch --all`
`git checkout [branch-name]`
# Checklist:
- x My code follows the style guidelines of this project
- x I have performed a self-review of my own code
- x My changes generate no new warnings
- x I have added test instructions that prove my fix is effective or that my feature works
